public/index.html

```diff
@@ -219,13 +235,9 @@ <h2>Schedules</h2>
             <form id="scheduleForm" class="schedule-form">
                 <h3>Add New Schedule</h3>
                 <div class="form-group">
-                    <label for="scheduleName">Namespace:</label>
-                    <input
-                        type="text"
-                        id="scheduleName"
-                        required
-                        placeholder="Enter namespace name"
-                    />
+                    <label for="scheduleName">Namespaces:</label>
+                    <select id="scheduleName" multiple required>
```

Comment:
what is 'multiple' for? The API only takes one at a time

Reply:
Wait for it